awx_collection/plugins/module_utils/tower_api.py

```diff
@@ -272,7 +272,7 @@ def make_request(self, method, endpoint, *args, **kwargs):
             if self._COLLECTION_TYPE not in self.collection_to_version or self.collection_to_version[self._COLLECTION_TYPE] != tower_type:
                 self.warn("You are using the {0} version of this collection but connecting to {1}".format(self._COLLECTION_TYPE, tower_type))
             elif collection_compare_ver != tower_compare_ver:
-                self.warn("You are running collection version {0} but connecting to tower version {1}".format(self._COLLECTION_VERSION, tower_version))
+                self.warn("You are running collection version {0} but connecting to {2} version {1}".format(self._COLLECTION_VERSION, tower_version, tower_type))
```

Comment:
Noticed (while running the test playbook) that the product name was hardcoded into the warning, thus this change.